### PR TITLE
[Merged by Bors] - chore(RingTheory.Polynomial.Basic): clean up porting notes

### PR DESCRIPTION
**Pull request by mattrobball**
Lean is so much more ergonomic now.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Polynomial/Basic.lean [hunk 1]
```diff
@@ -807,8 +807,7 @@ theorem is_fg_degreeLE [IsNoetherianRing R] (I : Ideal R[X]) (n : ℕ) :
     Submodule.FG (I.degreeLE n) :=
   letI := Classical.decEq R
   isNoetherian_submodule_left.1
-    -- porting note: times out without explicit `R`.
-    (isNoetherian_of_fg_of_noetherian _ ⟨_, (degreeLE_eq_span_X_pow (R := R)).symm⟩) _
+    (isNoetherian_of_fg_of_noetherian _ ⟨_, (degreeLE_eq_span_X_pow).symm⟩) _
```
Comment (Ruben-VandeVelde) [inline Mathlib/RingTheory/Polynomial/Basic.lean:810]:
```suggestion
    (isNoetherian_of_fg_of_noetherian _ ⟨_, degreeLE_eq_span_X_pow.symm⟩) _
```

File: Mathlib/RingTheory/Polynomial/Basic.lean [hunk 2]
```diff
@@ -878,11 +864,10 @@ theorem prime_C_iff : Prime (C r : MvPolynomial σ R) ↔ Prime r :=
       fun a b hd => by
       obtain ⟨s, a', b', rfl, rfl⟩ := exists_finset_rename₂ a b
       rw [← algebraMap_eq] at hd
-      have := (@killCompl s σ R _ ((↑) : s → σ) Subtype.coe_injective).toRingHom.map_dvd hd
-      have : algebraMap R _ r ∣ a' * b' := by convert this <;> simp
+      have : algebraMap R _ r ∣ a' * b' := by
+        convert killCompl Subtype.coe_injective|>.toRingHom.map_dvd hd <;> simp
```
Comment (Ruben-VandeVelde) [inline Mathlib/RingTheory/Polynomial/Basic.lean:868]: I think we put a space before `|>.`
```suggestion
        convert killCompl Subtype.coe_injective |>.toRingHom.map_dvd hd <;> simp
```

**Review by Ruben-VandeVelde — COMMENTED** [review]
Nice. Maybe you should `!bench` this?

File: Mathlib/RingTheory/Polynomial/Basic.lean (hunk 1 above)
Comment (mattrobball) [reply Mathlib/RingTheory/Polynomial/Basic.lean]: Done

File: Mathlib/RingTheory/Polynomial/Basic.lean (hunk 2 above)
Comment (mattrobball) [reply Mathlib/RingTheory/Polynomial/Basic.lean]: Done

**Review by jcommelin — COMMENTED** [review]
Wonderful!

bors merge

**Comment by Ruben-VandeVelde** [comment]
maintainer merge

**Comment by fpvandoorn** [comment]
bors merge

**Pull request closed without merging**